test/extended/cli/statefulset.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"os"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc statefulset", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc = exutil.NewCLI("oc-statefulset")
+	)
+
+	g.It("creates and deletes statefulsets", func() {
+		g.By("creating a new service for the statefulset")
+
+		frontendFile, err := writeObjectToFile(newFrontendService())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer os.Remove(frontendFile)
+
+		statefulSetFile, err := writeObjectToFile(newBusyBoxStatefulSet())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer os.Remove(statefulSetFile)
+
+		err = oc.Run("create").Args("-f", frontendFile).Execute()
```

Comment:
btw you can also use `InputString` to read the data from stdin instead of a file

Reply:
also can you please use the oc CLI client instead of an direct API calls in other places as well?

Reply:
those can be done in followups